pkg/minikube/machine/cache_images.go

```diff
@@ -83,10 +83,12 @@ func LoadImages(cc *config.ClusterConfig, runner command.Runner, images []string
 		return errors.Wrap(err, "runtime")
 	}
 
-	imgClient, err := client.NewClientWithOpts(client.FromEnv) // image client
-	if err != nil {
-		glog.Infof("couldn't get a local image daemon which might be ok: %v", err)
-		imgClient = nil
+	var imgClient *client.Client
+	if cr.Name() == "Docker" {
+		imgClient, err = client.NewClientWithOpts(client.FromEnv) // image client
+		if err != nil {
+			glog.Infof("couldn't get a local image daemon which might be ok: %v", err)
```

Comment:
do you still want to set it as nil ? 
imgClient = nil
or that one is not needed anymore ?

Reply:
It is still set as nil, if it cannot connect to the docker daemon (for whatever reason)...

The main change is that now I also set it to nil (don't ask docker), when using podman.

Reply:
As far as I know, this docker client is only used to query for image ID

Reply:
For the other runtimes, we use the stupid way of doing save and readning the manifest back

Reply:
I'm not sure that asking the docker daemon makes sense for anything but driver "none" anyway ?